components/alert/sidebar.css

```diff
@@ -0,0 +1,40 @@
+*{
```

Comment:
you can put it in your index css so you don't have to write it again and again.

Reply:
Okay . I'll read about it and update in what will correct way to import file.

Reply:
Moved it  to index.css Please look into it